src/test/kotlin/no/skatteetaten/aurora/boober/feature/SecretVaultFeatureTest.kt

```diff
@@ -186,7 +186,7 @@ class SecretVaultFeatureTest : AbstractFeatureTest() {
 
         val attachmentResource = resource.last()
         assertThat(attachmentResource.resource.metadata.name).isEqualTo(attachmentResource.resource.metadata.name.toLowerCase())
```

Comment:
Denne asserten her er kanskje litt farlig? Den sjekker jo bare at de to er like, men ikke at det navnet som er her faktisk er et gyldig navn for en kubernetes ressurs.